tool/gravity/cli/rotate.go

```diff
@@ -38,16 +39,47 @@ import (
 	"github.com/gravitational/trace"
 )
 
-type rotateOptions struct {
-	// clusterName is the local cluster name
-	clusterName string
-	// validFor specifies validity duration for renewed certs
-	validFor time.Duration
-	// caPath is optional CA to use
-	caPath string
+func rotateRPCCredentials(env *localenv.LocalEnvironment, o rotateRPCCredsOptions) (err error) {
+	clusterEnv, err := env.NewClusterEnvironment()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	archive, err := rpc.LoadCredentials(clusterEnv.Packages)
+	if err != nil && !trace.IsNotFound(err) {
+		return trace.Wrap(err)
+	}
+	now := time.Now()
+	if o.dryRun {
+		if archive == nil {
+			trace.NotFound("no RPC credentials found. Run the command again without --dry-run to generate.")
+		}
+		if err := rpc.ValidateCredentials(archive, now); err != nil {
+			return trace.Wrap(err)
+		}
+		env.Println("Nothing to do.")
+		return nil
+	}
+	if !o.force {
```

Comment:
I don't understand why we want/need force logic here? If a user requests rotation of the rpc credentials, why not just rotate them regardless of whether they're valid or not?